README.md

```diff
@@ -43,24 +44,26 @@ python setup.py install
 ```
 
 Some of the examples included in the MembraneCurvature documentation use test
-cases from [MDAnalysisTests]. To install the unit tests via conda:
+cases from [MDAnalysisTests]. To install the unit tests via `conda`:
 
 ```
 conda install -c conda-forge MDAnalysisTests
 ```
 
-or via pip:
+or via `pip`:
 
 ```
 pip install --upgrade MDAnalysisTests
 ```
 
+> ⚠️ In comparison to the previous version, `membrane-curvature==0.0.3` shows a significant improvement in performance, particularly notable for membrane-protein systems. Installing the last available version is highly encouraged. 
+
```

Comment:
Now that I read this section again I wonder if I should clarify that the package is available for Python 3.10 as well. I never said it explicitly, neither here nor in docs. What do you think @hmacdope? Maybe adding it at the beginning of the installation section?

Reply:
Its in the changelog in this PR, so i think its ok.